pure_changelog.md

```diff
@@ -49,6 +49,7 @@
 * Fixed broken events counter on main screen for more than 99 events
 * Fixed incorrect signal strength displayed in Center
 * Fixed inability to enter text in search field after clicking right arrow
+* Fixed for unexpected behavior when removing and importing contacts from SIM card more than once
```

Comment:
Instead of just _unexpected behavior_, please briefly write what was the problem (this one is the reason why I "requested changes" on this PR)